packages/application-shell/src/components/fetch-project/fetch-project.spec.js

```diff
@@ -44,20 +33,71 @@ const renderProject = (options) =>
   );
 
 describe('rendering', () => {
-  it('should fetch project and pass data to children function', async () => {
-    renderProject();
-    await waitForElementToBeRemoved(() => screen.getByText('loading...'));
-    expect(screen.getByText(/Test 1/i)).toBeInTheDocument();
+  describe('when fetching project succeeds', () => {
+    it('should fetch project and pass data to children function', async () => {
+      mockServer.use(
+        graphql.query('FetchProject', (req, res, ctx) =>
+          res.once(
```

Comment:
If you're doing it like this (define the mock for each test), there is no need to use `res.once`.

Using `res.once` is only useful if you have multiple mocks for the same query in the same test.
After each test, the mocks are reset.

Reply:
Yes, but both mocks operate on the same query. So I want to make sure to respond with one first once and then the other second. 

Can you maybe commit the change your suggesting? I don't fully understand the change request being made.

Reply:
Sure, let me try

Reply:
Please have a look: https://github.com/commercetools/merchant-center-application-kit/pull/1754/commits/19659837f407a3ac356854342a2f35ca65dcd9d8

Reply:
Got cha. Still not to fond of global setup or mocks but that is subjective:)